docs/user-guide/upgrade.md

```diff
@@ -4,7 +4,42 @@
 
 ## Release 2.0.0-alpha (not released yet)
 
-With release 2.0.0 we will use .NET 6 for the starter kit. Our docker images will be based on Debian 11 (Bullseye). Please make sure that if you plan to use our Docker images, you use Debian 11 instead of Debian 10. If you plan to use a Debian 10-based OS, you will need to build a custom Docker image.
+With release 2.0.0 we will use .NET 6 for the starter kit. Our docker images will be based on Debian 11 (Bullseye). Please make sure that if you plan to use our Docker images, you use a Debian 11-based OS instead of Debian 10. If you plan to use a Debian 10-based OS, you will need to build a custom Docker image.
+
+### Upgrading to Raspberry Pi OS (bullseye)
+
+If you are running IoT Edge on a Raspberry Pi that is based on Debian 10 (Buster), we recommend that you download a new version of the image and perform a clean install.
+
+After you installed the latest version of Raspberry Pi OS, execute the following commands to install IoT Edge:
+
+```bash
+curl https://packages.microsoft.com/config/debian/11/packages-microsoft-prod.deb > ./packages-microsoft-prod.deb
+sudo apt-get install ./packages-microsoft-prod.deb
+sudo apt-get update
+sudo apt-get install moby-engine
+curl -L https://github.com/Azure/azure-iotedge/releases/download/1.2.5/aziot-identity-service_1.2.4-1_debian11_armhf.deb -o aziot-identity-service.deb && sudo apt-get install ./aziot-identity-service.deb
```

Comment:
I would say this example is for iot edge 1.2.5, people that want another version can follow instructions at https://github.com/Azure/iotedge-lorawan-starterkit/pull/953/files

Reply:
I agree, but i think you put the wrong link https://docs.microsoft.com/en-us/azure/iot-edge/how-to-provision-single-device-linux-symmetric?view=iotedge-2020-11&tabs=azure-portal#offline-or-specific-version-installation-optional

Reply:
Done with ec476229b8fe851f2d1fa93f29ee9ad4f8a1a666